core/config/dev.exs

```diff
@@ -77,10 +77,6 @@ config :core, Systems.Email.Mailer,
 
 config :core, :apns_backend, Core.APNS.LoggingBackend
 
-config :core,
-       :admins,
-       ["e.vanderveen@eyra.co", "m.j.kokernoot@eyra.co"]
-
```

Comment:
Was deze intended?

Reply:
Ja. Boven in de de dev config staat al een asterisk voor iedereen met een eyra email adres.